src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -859,6 +875,16 @@ private TerminalKeyInfo(TermInfo.Database db)
                     AddKey(db, TermInfo.Database.KeySRight, ConsoleKey.RightArrow, true, false, false);
                     AddKey(db, TermInfo.Database.KeyUp, ConsoleKey.UpArrow);
 
+                    AddPrefixKey(db, "kLFT", ConsoleKey.LeftArrow);
```

Comment:
nit: Should we make these identifiers constants? I suppose it's obvious what they refer to by the enum following them.